api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -530,7 +530,14 @@ public List<ReportingWorkspace> getWorkspaceBatch(long limit, long offset) {
             + "  a.short_name AS access_tier_short_name,\n"
             + "  billing_account_name,\n"
             + "  CASE \n"
-            + "    WHEN (w.billing_account_name = ? AND (w.initial_credits_exhausted = 1 OR w.initial_credits_expired = 1)) THEN ? \n"
+            + "    WHEN (w.billing_account_name = ? AND\n"
+            + "      (uvia.institution_id IS NULL OR\n"
```

Comment:
Handle case where users do not have an institution. Set their billing status to inactive.